components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java

```diff
@@ -785,7 +785,7 @@ public Response updateAPI(String apiId, APIDTO body, String ifMatch, MessageCont
             originalAPI.setOrganization(organization);
             //validate API update operation permitted based on the LC state
             validateAPIOperationsPerLC(originalAPI.getStatus());
-            API updatedApi = PublisherCommonUtils.updateApi(originalAPI, body, apiProvider, tokenScopes);
+            API updatedApi = PublisherCommonUtils.updateApi(originalAPI, body, apiProvider, tokenScopes, organization);
```

Comment:
we don't need organization here as well

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/10621/commits/d711021b49bdfd26cece08490dee2dcd69b5d626